docs/source/run.rst

```diff
@@ -1,7 +1,26 @@
+===============
 Running the bot
 ===============
 
+Welcome to this part of the documentation, dedicated to all who do or want to
+run a Sopel instance. This guide will help you set up your instance and manage
+your plugins and your configuration.
+
+.. note::
+
+    Sopel tries to be as simple as possible, however "simple" can be very
+    subjective, so do not hesitate to ask questions to the maintainers on IRC
+    (on the :ref:`#sopel channel<faq-contact-us>` on libera.chat) or through
+    Sopel's `GitHub repository`__. This guide is far from perfect, and we
+    appreciate feedback to make it better for everyone.
+
 .. toctree::
+    :titlesonly:
+
+    run/install
+    run/plugin
+    run/service
+    run/cli
+    run/configuration
 
-    cli
-    configuration
+.. __: https://github.com/sopel-irc/sopel/
```

Comment:
Extremely nitpicky; GH doesn't even bother redirecting this:

```suggestion
.. __: https://github.com/sopel-irc/sopel
```

Reply:
The most nitpicky of nitpick History. 😁